konfera/models/ticket.py

```diff
@@ -53,6 +52,8 @@ def clean(self):
             exist_ticket = self.order.ticket_set.first()
             if exist_ticket and exist_ticket.type.event.id != self.type.event.id:
                 raise ValidationError(_('All tickets must be for the same event.'))
+        if self.discount_code and self.discount_code.is_available is False:
```

Comment:
This is valid code, but I think if you change it to:
`if self.discount_code and not self.discount_code.is_available:`
it would be more 'pythonic'...